lightning/src/routing/router.rs

```diff
@@ -1251,8 +1251,8 @@ where L::Target: Logger {
 			|info| info.features.supports_basic_mpp()))
 	} else { false };
 
-	log_trace!(logger, "Searching for a route from payer {} to payee {:?} {} MPP and {} first hops {}overriding the network graph", our_node_pubkey,
-		payment_params.payee, if allow_mpp { "with" } else { "without" },
+	log_trace!(logger, "Searching for a route from payer {} to payee {} {} MPP and {} first hops {}overriding the network graph", our_node_pubkey,
+		maybe_dummy_payee_pk, if allow_mpp { "with" } else { "without" },
```

Comment:
Should the logging distinguish a dummy payee id?

Reply:
Yeah, you're right. Will fix